yellowbrick/classifier/class_prediction_error.py

```diff
@@ -1,194 +1,33 @@
-# yellowbrick.classifier.class_balance
-# Class balance visualizer for showing per-class support.
+# yellowbrick.classifier.class_prediction_error
+# Shows the balance of classes and their associated predictions.
 #
-# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
-# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
-# Author:   Neal Humphrey
 # Author:   Larry Gray
+# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
 # Created:  Wed May 18 12:39:40 2016 -0400
 #
-# Copyright (C) 2017 District Data Labs
-# For license information, see LICENSE.txt
-#
-# ID: class_balance.py [5388065] neal@nhumphrey.com $
+# ID: class_prediction_error.py [] lwgray@gmail.com $
 
 """
-Class balance visualizer for showing per-class support.
+Shows the balance of classes and their associated predictions.
 """
 
 ##########################################################################
 ## Imports
 ##########################################################################
 
-import matplotlib.pyplot as plt
 import numpy as np
+import matplotlib.pyplot as plt
 
 from .base import ClassificationScoreVisualizer
 
-from sklearn.model_selection import train_test_split
-from sklearn.metrics import precision_recall_fscore_support
 from sklearn.utils.multiclass import unique_labels
 from sklearn.metrics.classification import _check_targets
+from sklearn.model_selection import train_test_split as tts
 
 from ..exceptions import ModelError, YellowbrickValueError
 from ..style.colors import resolve_colors
 
 
-##########################################################################
-## Class Balance Chart
-##########################################################################
-
-class ClassBalance(ClassificationScoreVisualizer):
```

Comment:
Most of the changes here are removing `ClassBalance` and keeping `ClassPredictionError`